cve_bin_tool/cli.py

```diff
@@ -253,20 +255,22 @@ def extract_file(ectx, walker, should_extract, filepath, file_list):
     if ectx.can_extract(filepath):
         if not should_extract:
             file_list.append(filepath)
-            print("%s is an archive. Pass " % (filepath,) + "-x option to auto-extract")
+            LOGGER.debug(
```

Comment:
This one should also be a warning rather than debug message.  (as in, we probably want people to see this message during normal use, which would probably be at "info" level rather than "debug" level.  Although I think we might be defaulting to "debug" and should proably fix that too.)